python/pywatchman/bser.c

```diff
@@ -1084,9 +1087,87 @@ static PyObject *bser_loads(PyObject *self, PyObject *args, PyObject *kw)
   return bser_loads_recursive(&data, end, &ctx);
 }
 
+static PyObject *bser_load(PyObject *self, PyObject *args, PyObject *kw)
+{
+  size_t rc = 0;
+  FILE *fp = NULL;
+  char header[SNIFF_BUFFER_SIZE];
+  const char *data = NULL;
+  char *newdata = NULL;
+  const char *end;
+  int64_t expected_len, total_len;
+  off_t position;
+  PyObject *fp_obj = NULL;
+  PyObject *mutable_obj = NULL;
+  const char *value_encoding = NULL;
+  const char *value_errors = NULL;
+  unser_ctx_t ctx = {1, 0};
+
+  PyObject *ret = NULL;
+
+  static char *kw_list[] = {"fp", "mutable", "value_encoding", "value_errors",
```

Comment:
I realize that this is just copy-pasted from above, but can this be made `const`?